Support/Multitenancy/Ironic_bmo-mult-instances/03-images-and-run-local-services.sh

```diff
@@ -83,7 +83,7 @@ cat <<'EOF' > /opt/metal3-dev-env/ironic/virtualbmc/sushy-tools/htpasswd
 admin:$2b$12$/dVOBNatORwKpF.ss99KB.vESjfyONOxyH.UgRwNyZi1Xs/W2pGVS
 EOF
 ssh-keygen -f /root/.ssh/id_rsa_virt_power -P ""
-cat /root/.ssh/id_rsa_virt_power.pub | tee -a /root/.ssh/authorized_keys
+tee -a /root/.ssh/authorized_keys < /root/.ssh/id_rsa_virt_power.pub
```

Comment:
It works like this, alternatively you could just `cat /root/.ssh/id_rsa_virt_power.pub >> /root/.ssh/authorized_keys`.

`tee -a` trick is usually only used when dealing with `sudo` as redirection occurs before privilege elevation so `>>` doesn't work.